### PR TITLE
feat: add create customer group

### PR DESCRIPTION
**Pull request by yusufttur**

### CODE REVIEW

File: src/page-objects/StorefrontPages.ts [hunk 1]
```diff
@@ -53,7 +55,7 @@ export const StorefrontPageObjects = {
     AccountAddresses,
     AccountPayment,
     Search,
-    SearchSuggest,
```
Comment (vanpham-sw) [inline src/page-objects/StorefrontPages.ts:56]: We don't want to delete this SearchSuggest right?

Comment (yusufttur) [reply src/page-objects/StorefrontPages.ts:56]: It is my mistake, fixed 🙈

File: src/page-objects/administration/CustomerDetail.ts [hunk 2]
```diff
@@ -16,18 +19,22 @@ export class CustomerDetail implements PageObject {
         this.customFieldCard = page.locator('.sw-card').getByText('Custom fields');
         this.customFieldSetTabs = this.customFieldCard.locator('.sw-tabs-item');
         this.customFieldSetTabCustomContent = this.customFieldCard.locator('.sw-tabs__custom-content');
+        this.customerGroupRequestMessage = page.getByText(/Access to customer group ".*" requested\./);
```
Comment (vanpham-sw) [inline src/page-objects/administration/CustomerDetail.ts]: I would get this locator dynamically if possible if we want to check the entire text, [see here](https://github.com/shopware/acceptance-test-suite/commit/f079431044175f416ca1457d0a9fda91f79d2d27#diff-6df0c5e7cb97654b2850e125bbfb6e9e8804c9b8e41a5715cd79d91beed917e1R88) wdyt?

File: src/page-objects/storefront/Account.ts [hunk 3]
```diff
@@ -18,6 +19,7 @@ export class Account implements PageObject {
         this.shippingAddressCardTitle = page.getByRole('heading', { name: 'Default shipping address' });
         this.newsletterCheckbox = page.getByLabel('Yes, I would like to');
         this.newsletterRegistrationSuccessMessage = page.getByText('You have successfully subscribed to the newsletter.');
+        this.customerGroupRequestMessage = page.getByText(/Access to customer group ".*" requested\./);
```
Comment (vanpham-sw) [inline src/page-objects/storefront/Account.ts]: I would get this locator dynamically if possible if we want to check the entire text, [see here](https://github.com/shopware/acceptance-test-suite/commit/f079431044175f416ca1457d0a9fda91f79d2d27#diff-6df0c5e7cb97654b2850e125bbfb6e9e8804c9b8e41a5715cd79d91beed917e1R88) wdyt?

File: src/services/TestDataService.ts [hunk 4]
```diff
@@ -1940,4 +1941,44 @@ export class TestDataService {
...
+        
+        const basicCustomerGroup = this.getBasicCustomerGroup(overrides);
+
+        const response = await this.AdminApiClient.post('customer-group?_response=detail', {
+            data: basicCustomerGroup,
+        });
+
+        expect(response.ok()).toBeTruthy();
+
+        const { data: CustomerGroup } = (await response.json()) as { data: CustomerGroup };
```
Comment (vanpham-sw) [inline src/services/TestDataService.ts]:
```suggestion
        const { data: customerGroup } = (await response.json()) as { data: CustomerGroup };
```

File: src/services/TestDataService.ts [hunk 5]
```diff
@@ -1940,4 +1941,44 @@ export class TestDataService {
...
+
+        this.addCreatedRecord('customer_group', CustomerGroup.id);
+
+        return CustomerGroup;
```
Comment (vanpham-sw) [inline src/services/TestDataService.ts]:
```suggestion
        return customerGroup;
```

File: src/page-objects/administration/CustomerDetail.ts (hunk 2 above)
Comment (yusufttur) [reply src/page-objects/administration/CustomerDetail.ts]: I applied a helper method.

File: src/page-objects/storefront/Account.ts (hunk 3 above)
Comment (yusufttur) [reply src/page-objects/storefront/Account.ts]: I applied a helper method.

**Pull request closed without merging**